From 32fb521ed246ffd8f339b8e3ba2719e2968205d9 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <heinrich.schuchardt@canonical.com>
Date: Thu, 1 Feb 2024 22:24:28 +0100
Subject: [PATCH edk2-test 1/1] uefi-sct/SctPkg: correct %LX SctPrint code

REF: https://bugzilla.tianocore.org/show_bug.cgi?id=4670

SctPrint() uses 'l' to indicate a UINT64 number. It does not support 'L'.
Replace all occurrences of %LX by %lX.

Signed-off-by: Heinrich Schuchardt <heinrich.schuchardt@canonical.com>
---
 .../MemoryAllocationServicesBBTestFunction.c  |  4 +-
 .../PciRootBridgeIoBBTestConformance.c        | 78 +++++++++----------
 .../PciRootBridgeIoBBTestFunction_1.c         | 28 +++----
 .../PciRootBridgeIoBBTestFunction_2.c         | 28 +++----
 .../PciRootBridgeIoBBTestStress.c             |  8 +-
 .../PciRootBridgeIoBBTestSupport.c            |  2 +-
 6 files changed, 74 insertions(+), 74 deletions(-)

diff --git a/uefi-sct/SctPkg/TestCase/UEFI/EFI/BootServices/MemoryAllocationServices/BlackBoxTest/MemoryAllocationServicesBBTestFunction.c b/uefi-sct/SctPkg/TestCase/UEFI/EFI/BootServices/MemoryAllocationServices/BlackBoxTest/MemoryAllocationServicesBBTestFunction.c
index 4e65d7a1c562..57f06358fd0d 100644
--- a/uefi-sct/SctPkg/TestCase/UEFI/EFI/BootServices/MemoryAllocationServices/BlackBoxTest/MemoryAllocationServicesBBTestFunction.c
+++ b/uefi-sct/SctPkg/TestCase/UEFI/EFI/BootServices/MemoryAllocationServices/BlackBoxTest/MemoryAllocationServicesBBTestFunction.c
@@ -553,7 +553,7 @@ BBTestAllocatePagesInterfaceTest (
                         gMemoryAllocationServicesBBTestFunctionAssertionGuid021: \
                         gMemoryAllocationServicesBBTestFunctionAssertionGuid022),
                        L"BS.AllocatePages - AllocateMaxAddress return Address",
-                       L"%a:%d:TPL - %d, MemType - %d, PhysicalStart - 0x%LX, NumberOfPages - 0x%LX, Memory - 0x%LX",
+                       L"%a:%d:TPL - %d, MemType - %d, PhysicalStart - 0x%lX, NumberOfPages - 0x%lX, Memory - 0x%lX",
                        __FILE__,
                        (UINTN)__LINE__,
                        TplArray[Index],
@@ -630,7 +630,7 @@ BBTestAllocatePagesInterfaceTest (
                         gMemoryAllocationServicesBBTestFunctionAssertionGuid027: \
                         gMemoryAllocationServicesBBTestFunctionAssertionGuid028),
                        L"BS.AllocatePages - AllocateMaxAddress return Address2",
-                       L"%a:%d:TPL - %d, MemType - %d, Memory2 - 0x%LX",
+                       L"%a:%d:TPL - %d, MemType - %d, Memory2 - 0x%lX",
                        __FILE__,
                        (UINTN)__LINE__,
                        TplArray[Index],
diff --git a/uefi-sct/SctPkg/TestCase/UEFI/EFI/Protocol/PciRootBridgeIo/BlackBoxTest/PciRootBridgeIoBBTestConformance.c b/uefi-sct/SctPkg/TestCase/UEFI/EFI/Protocol/PciRootBridgeIo/BlackBoxTest/PciRootBridgeIoBBTestConformance.c
index c21f42745ca6..4e6dc7aa23dc 100644
--- a/uefi-sct/SctPkg/TestCase/UEFI/EFI/Protocol/PciRootBridgeIo/BlackBoxTest/PciRootBridgeIoBBTestConformance.c
+++ b/uefi-sct/SctPkg/TestCase/UEFI/EFI/Protocol/PciRootBridgeIo/BlackBoxTest/PciRootBridgeIoBBTestConformance.c
@@ -848,7 +848,7 @@ MemRead_Conf (
                  AssertionType,
                  gPciRootBridgeIoBBTestConformanceAssertionGuid011,
                  L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.Mem.Read() - With Width as EfiPciWidthMaximum the return Status must be EFI_INVALID_PARAMETER",
-                 L"%a:%d:Status - %r,Width -%s,Address -%LXh",
+                 L"%a:%d:Status - %r,Width -%s,Address -%lXh",
                  __FILE__,
                  (UINTN)__LINE__,
                  Status,
@@ -882,7 +882,7 @@ MemRead_Conf (
                  AssertionType,
                  gPciRootBridgeIoBBTestConformanceAssertionGuid012,
                  L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.Mem.Read() - With Width as -1 the return Status must be EFI_INVALID_PARAMETER",
-                 L"%a:%d:Status - %r,Width -%d,Address -%LXh",
+                 L"%a:%d:Status - %r,Width -%d,Address -%lXh",
                  __FILE__,
                  (UINTN)__LINE__,
                  Status,
@@ -917,7 +917,7 @@ MemRead_Conf (
                  AssertionType,
                  gPciRootBridgeIoBBTestConformanceAssertionGuid013,
                  L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.Mem.Read() - With Result as NULL the return Status must be EFI_INVALID_PARAMETER",
-                 L"%a:%d:Status - %r,Width -%s,Address -%LXh",
+                 L"%a:%d:Status - %r,Width -%s,Address -%lXh",
                  __FILE__,
                  (UINTN)__LINE__,
                  Status,
@@ -1056,7 +1056,7 @@ MemRead_Conf (
                    AssertionType,
                    gPciRootBridgeIoBBTestConformanceAssertionGuid014,
                    L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.Mem.Read() - With invalid Width the return Status must be EFI_INVALID_PARAMETER",
-                   L"%a:%d:Status - %r,Width -%s,Address -%LXh",
+                   L"%a:%d:Status - %r,Width -%s,Address -%lXh",
                    __FILE__,
                    (UINTN)__LINE__,
                    Status,
@@ -1224,7 +1224,7 @@ MemWrite_Conf (
                  AssertionType,
                  gPciRootBridgeIoBBTestConformanceAssertionGuid015,
                  L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.Mem.Write() - With Width as EfiPciWidthMaximum the return Status must be EFI_INVALID_PARAMETER",
-                 L"%a:%d:Status - %r,Width -%s,Address -%LXh",
+                 L"%a:%d:Status - %r,Width -%s,Address -%lXh",
                  __FILE__,
                  (UINTN)__LINE__,
                  Status,
@@ -1258,7 +1258,7 @@ MemWrite_Conf (
                  AssertionType,
                  gPciRootBridgeIoBBTestConformanceAssertionGuid016,
                  L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.Mem.Write() - With Width as -1 the return Status must be EFI_INVALID_PARAMETER",
-                 L"%a:%d:Status - %r,Width -%d,Address -%LXh",
+                 L"%a:%d:Status - %r,Width -%d,Address -%lXh",
                  __FILE__,
                  (UINTN)__LINE__,
                  Status,
@@ -1431,7 +1431,7 @@ MemWrite_Conf (
                    AssertionType,
                    gPciRootBridgeIoBBTestConformanceAssertionGuid018,
                    L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.Mem.Write() - With invalid Width the return Status must be EFI_INVALID_PARAMETER",
-                   L"%a:%d:Status - %r,Width -%s,Address -%LXh",
+                   L"%a:%d:Status - %r,Width -%s,Address -%lXh",
                    __FILE__,
                    (UINTN)__LINE__,
                    Status,
@@ -1597,7 +1597,7 @@ IoRead_Conf (
                  AssertionType,
                  gPciRootBridgeIoBBTestConformanceAssertionGuid019,
                  L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.Io.Read() - With Width as EfiPciWidthMaximum the return Status must be EFI_INVALID_PARAMETER",
-                 L"%a:%d:Status - %r,Width -%s,Address -%LXh",
+                 L"%a:%d:Status - %r,Width -%s,Address -%lXh",
                  __FILE__,
                  (UINTN)__LINE__,
                  Status,
@@ -1631,7 +1631,7 @@ IoRead_Conf (
                  AssertionType,
                  gPciRootBridgeIoBBTestConformanceAssertionGuid020,
                  L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.Io.Read() - With Width as -1 the return Status must be EFI_INVALID_PARAMETER",
-                 L"%a:%d:Status - %r,Width -%d, Address -%LXh",
+                 L"%a:%d:Status - %r,Width -%d, Address -%lXh",
                  __FILE__,
                  (UINTN)__LINE__,
                  Status,
@@ -1666,7 +1666,7 @@ IoRead_Conf (
                  AssertionType,
                  gPciRootBridgeIoBBTestConformanceAssertionGuid021,
                  L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.Io.Read() - With Result as NULL the return Status must be EFI_INVALID_PARAMETER",
-                 L"%a:%d:Status - %r,Width -%s,Address -%LXh",
+                 L"%a:%d:Status - %r,Width -%s,Address -%lXh",
                  __FILE__,
                  (UINTN)__LINE__,
                  Status,
@@ -1804,7 +1804,7 @@ IoRead_Conf (
                    AssertionType,
                    gPciRootBridgeIoBBTestConformanceAssertionGuid022,
                    L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.Io.Read() - With invalid Width the return Status must be EFI_INVALID_PARAMETER",
-                   L"%a:%d:Status - %r,Width -%s,Address -%LXh",
+                   L"%a:%d:Status - %r,Width -%s,Address -%lXh",
                    __FILE__,
                    (UINTN)__LINE__,
                    Status,
@@ -1970,7 +1970,7 @@ IoWrite_Conf (
                  AssertionType,
                  gPciRootBridgeIoBBTestConformanceAssertionGuid023,
                  L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.Io.Write() - With Width as EfiPciWidthMaximum the return Status must be EFI_INVALID_PARAMETER",
-                 L"%a:%d:Status - %r,Width -%s,Address -%LXh",
+                 L"%a:%d:Status - %r,Width -%s,Address -%lXh",
                  __FILE__,
                  (UINTN)__LINE__,
                  Status,
@@ -2004,7 +2004,7 @@ IoWrite_Conf (
                  AssertionType,
                  gPciRootBridgeIoBBTestConformanceAssertionGuid024,
                  L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.Io.Write() - With Width as -1 the return Status must be EFI_INVALID_PARAMETER",
-                 L"%a:%d:Status - %r,Width -%d,Address -%LXh",
+                 L"%a:%d:Status - %r,Width -%d,Address -%lXh",
                  __FILE__,
                  (UINTN)__LINE__,
                  Status,
@@ -2039,7 +2039,7 @@ IoWrite_Conf (
                  AssertionType,
                  gPciRootBridgeIoBBTestConformanceAssertionGuid025,
                  L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.Io.Write() - With Result as NULL the return Status must be EFI_INVALID_PARAMETER",
-                 L"%a:%d:Status - %r,Width -%s,Address -%LXh",
+                 L"%a:%d:Status - %r,Width -%s,Address -%lXh",
                  __FILE__,
                  (UINTN)__LINE__,
                  Status,
@@ -2177,7 +2177,7 @@ IoWrite_Conf (
                    AssertionType,
                    gPciRootBridgeIoBBTestConformanceAssertionGuid026,
                    L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.Io.Write() - With invalid Width the return Status must be EFI_INVALID_PARAMETER",
-                   L"%a:%d:Status - %r,Width -%s,Address -%LXh",
+                   L"%a:%d:Status - %r,Width -%s,Address -%lXh",
                    __FILE__,
                    (UINTN)__LINE__,
                    Status,
@@ -2330,7 +2330,7 @@ PciRead_Conf (
                  AssertionType,
                  gPciRootBridgeIoBBTestConformanceAssertionGuid027,
                  L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.Pci.Read() - With Width as EfiPciWidthMaximum the return Status must be EFI_INVALID_PARAMETER",
-                 L"%a:%d:Status - %r,Width -%s,Address -%LXh",
+                 L"%a:%d:Status - %r,Width -%s,Address -%lXh",
                  __FILE__,
                  (UINTN)__LINE__,
                  Status,
@@ -2364,7 +2364,7 @@ PciRead_Conf (
                  AssertionType,
                  gPciRootBridgeIoBBTestConformanceAssertionGuid028,
                  L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.Pci.Read() - With Width as -1 the return Status must be EFI_INVALID_PARAMETER",
-                 L"%a:%d:Status - %r,Width -%d,Address -%LXh",
+                 L"%a:%d:Status - %r,Width -%d,Address -%lXh",
                  __FILE__,
                  (UINTN)__LINE__,
                  Status,
@@ -2399,7 +2399,7 @@ PciRead_Conf (
                  AssertionType,
                  gPciRootBridgeIoBBTestConformanceAssertionGuid029,
                  L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.Pci.Read() - With Result as NULL the return Status must be EFI_INVALID_PARAMETER",
-                 L"%a:%d:Status - %r,Width -%s,Address -%LXh",
+                 L"%a:%d:Status - %r,Width -%s,Address -%lXh",
                  __FILE__,
                  (UINTN)__LINE__,
                  Status,
@@ -2541,7 +2541,7 @@ PciRead_Conf (
                    AssertionType,
                    gPciRootBridgeIoBBTestConformanceAssertionGuid030,
                    L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.Pci.Read() - With invalid Width the return Status must be EFI_INVALID_PARAMETER",
-                   L"%a:%d:Status - %r,Width -%s,Address -%LXh",
+                   L"%a:%d:Status - %r,Width -%s,Address -%lXh",
                    __FILE__,
                    (UINTN)__LINE__,
                    Status,
@@ -2694,7 +2694,7 @@ PciWrite_Conf (
                  AssertionType,
                  gPciRootBridgeIoBBTestConformanceAssertionGuid031,
                  L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.Pci.Write() - With Width as EfiPciWidthMaximum the return Status must be EFI_INVALID_PARAMETER",
-                 L"%a:%d:Status - %r,Width -%s,Address -%LXh",
+                 L"%a:%d:Status - %r,Width -%s,Address -%lXh",
                  __FILE__,
                  (UINTN)__LINE__,
                  Status,
@@ -2728,7 +2728,7 @@ PciWrite_Conf (
                  AssertionType,
                  gPciRootBridgeIoBBTestConformanceAssertionGuid032,
                  L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.Pci.Write() - With Width as -1 the return Status must be EFI_INVALID_PARAMETER",
-                 L"%a:%d:Status - %r,Width -%d,Address -%LXh",
+                 L"%a:%d:Status - %r,Width -%d,Address -%lXh",
                  __FILE__,
                  (UINTN)__LINE__,
                  Status,
@@ -2763,7 +2763,7 @@ PciWrite_Conf (
                  AssertionType,
                  gPciRootBridgeIoBBTestConformanceAssertionGuid033,
                  L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.Pci.Write() - With Result as NULL the return Status must be EFI_INVALID_PARAMETER",
-                 L"%a:%d:Status - %r,Width -%s,Address -%LXh",
+                 L"%a:%d:Status - %r,Width -%s,Address -%lXh",
                  __FILE__,
                  (UINTN)__LINE__,
                  Status,
@@ -2905,7 +2905,7 @@ PciWrite_Conf (
                    AssertionType,
                    gPciRootBridgeIoBBTestConformanceAssertionGuid034,
                    L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.Pci.Write() - With invalid Width the return Status must be EFI_INVALID_PARAMETER",
-                   L"%a:%d:Status - %r,Width -%s,Address -%LXh",
+                   L"%a:%d:Status - %r,Width -%s,Address -%lXh",
                    __FILE__,
                    (UINTN)__LINE__,
                    Status,
@@ -3075,7 +3075,7 @@ CopyMem_Conf (
                  AssertionType,
                  gPciRootBridgeIoBBTestConformanceAssertionGuid035,
                  L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.CopyMem() - With Width as EfiPciWidthMaximum the return Status must be EFI_INVALID_PARAMETER",
-                 L"%a:%d:Status - %r,Width -%s,Src Address -%LXh,Dst Address -%LXh",
+                 L"%a:%d:Status - %r,Width -%s,Src Address -%lXh,Dst Address -%lXh",
                  __FILE__,
                  (UINTN)__LINE__,
                  Status,
@@ -3110,7 +3110,7 @@ CopyMem_Conf (
                  AssertionType,
                  gPciRootBridgeIoBBTestConformanceAssertionGuid036,
                  L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.CopyMem() - With Width as -1 the return Status must be EFI_INVALID_PARAMETER",
-                 L"%a:%d:Status - %r,Width -%d,Src Address -%LXh,Dst Address -%LXh",
+                 L"%a:%d:Status - %r,Width -%d,Src Address -%lXh,Dst Address -%lXh",
                  __FILE__,
                  (UINTN)__LINE__,
                  Status,
@@ -3145,7 +3145,7 @@ CopyMem_Conf (
                    StandardLib,
                    EFI_VERBOSE_LEVEL_DEFAULT,
                    L"PCI_ROOT_BRIDGE_IO_PROTOCOL.CopyMem -with invalid Width return status should be EFI_INVALID_PARAMETER.\n"
-                   L"%a:%d:Status - %r,Width -%s,Src Address -%LXh,Dst Address-%LXh",
+                   L"%a:%d:Status - %r,Width -%s,Src Address -%lXh,Dst Address-%lXh",
                    __FILE__,
                    (UINTN)__LINE__,
                    Status,
@@ -3203,7 +3203,7 @@ CopyMem_Conf (
                    StandardLib,
                    EFI_VERBOSE_LEVEL_DEFAULT,
                    L"PCI_ROOT_BRIDGE_IO_PROTOCOL.CopyMem -with invalid Width return status should be EFI_INVALID_PARAMETER.\n"
-                   L"%a:%d:Status - %r,Width -%s,Src Address -%LXh,Dst Address-%LXh",
+                   L"%a:%d:Status - %r,Width -%s,Src Address -%lXh,Dst Address-%lXh",
                    __FILE__,
                    (UINTN)__LINE__,
                    Status,
@@ -3364,7 +3364,7 @@ CopyMem_Conf (
                    AssertionType,
                    gPciRootBridgeIoBBTestConformanceAssertionGuid039,
                    L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.CopyMem() - With invalid Width the return Status must be EFI_INVALID_PARAMETER",
-                   L"%a:%d:Status - %r,Width -%s,Src Address -%LXh,Dst Address -%LXh",
+                   L"%a:%d:Status - %r,Width -%s,Src Address -%lXh,Dst Address -%lXh",
                    __FILE__,
                    (UINTN)__LINE__,
                    Status,
@@ -4225,7 +4225,7 @@ SetAttributes_Conf (
                    AssertionType,
                    gPciRootBridgeIoBBTestConformanceAssertionGuid059,
                    L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.SetAttributes - with unsupported attributes the return status must be EFI_UNSUPPORTED.",
-                   L"%a:%d:Status - %r,Set Attributes - %LXh,Supported Attribute -%LXh",
+                   L"%a:%d:Status - %r,Set Attributes - %lXh,Supported Attribute -%lXh",
                    __FILE__,
                    (UINTN)__LINE__,
                    Status,
@@ -4254,7 +4254,7 @@ SetAttributes_Conf (
                    AssertionType,
                    gTestGenericFailureGuid,
                    L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.SetAttributes - Set Unsupported Attributes the attributes should remain unchanged.",
-                   L"%a:%d:Attributes before Set -%LXh,Set Attributes -%LXh,Current Attributes -%LXh",
+                   L"%a:%d:Attributes before Set -%lXh,Set Attributes -%lXh,Current Attributes -%lXh",
                    __FILE__,
                    (UINTN)__LINE__,
                    CurrentAttributes,
@@ -4338,7 +4338,7 @@ SetAttributes_Conf (
                    AssertionType,
                    gPciRootBridgeIoBBTestConformanceAssertionGuid060,
                    L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.SetAttributes - with unsupported attributes the return status must be EFI_UNSUPPORTED.",
-                   L"%a:%d:Status - %r,Set Attributes - %LXh,Supported Attribute -%LXh",
+                   L"%a:%d:Status - %r,Set Attributes - %lXh,Supported Attribute -%lXh",
                    __FILE__,
                    (UINTN)__LINE__,
                    Status,
@@ -4367,7 +4367,7 @@ SetAttributes_Conf (
                    AssertionType,
                    gTestGenericFailureGuid,
                    L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.SetAttributes - Set Unsupported Attributes the attributes should remain unchanged.",
-                   L"%a:%d:Attributes before Set -%LXh,Set Attributes -%LXh,Current Attributes -%LXh",
+                   L"%a:%d:Attributes before Set -%lXh,Set Attributes -%lXh,Current Attributes -%lXh",
                    __FILE__,
                    (UINTN)__LINE__,
                    CurrentAttributes,
@@ -4419,7 +4419,7 @@ SetAttributes_Conf (
                    AssertionType,
                    gPciRootBridgeIoBBTestConformanceAssertionGuid061,
                    L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.SetAttributes - with more than one attributes that need a resource the return status must be EFI_INVALID_PARAMETER.",
-                   L"%a:%d:Status - %r,Set Attributes - %LXh,Require Resource Attributes -%LXh,Supported Attribute -%LXh",
+                   L"%a:%d:Status - %r,Set Attributes - %lXh,Require Resource Attributes -%lXh,Supported Attribute -%lXh",
                    __FILE__,
                    (UINTN)__LINE__,
                    Status,
@@ -4463,7 +4463,7 @@ SetAttributes_Conf (
                    AssertionType,
                    gPciRootBridgeIoBBTestConformanceAssertionGuid062,
                    L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.SetAttributes - MEMORY_WRITE_COMBINE with ResourceBase as NULL the return status must be EFI_INVALID_PARAMETER.",
-                   L"%a:%d:Status - %r:Set Attributes - %LXh",
+                   L"%a:%d:Status - %r:Set Attributes - %lXh",
                    __FILE__,
                    (UINTN)__LINE__,
                    Status,
@@ -4495,7 +4495,7 @@ SetAttributes_Conf (
                    AssertionType,
                    gPciRootBridgeIoBBTestConformanceAssertionGuid063,
                    L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.SetAttributes - MEMORY_WRITE_COMBINE with ResourceLength as NULL the return status must be EFI_INVALID_PARAMETER.",
-                   L"%a:%d:Status - %r:Set Attributes - %LXh",
+                   L"%a:%d:Status - %r:Set Attributes - %lXh",
                    __FILE__,
                    (UINTN)__LINE__,
                    Status,
@@ -4531,7 +4531,7 @@ SetAttributes_Conf (
                    AssertionType,
                    gPciRootBridgeIoBBTestConformanceAssertionGuid064,
                    L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.SetAttributes - MEMORY_CACHED with ResourceBase as NULL the return status must be EFI_INVALID_PARAMETER.",
-                   L"%a:%d:Status - %r:Set Attributes - %LXh",
+                   L"%a:%d:Status - %r:Set Attributes - %lXh",
                    __FILE__,
                    (UINTN)__LINE__,
                    Status,
@@ -4563,7 +4563,7 @@ SetAttributes_Conf (
                    AssertionType,
                    gPciRootBridgeIoBBTestConformanceAssertionGuid065,
                    L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.SetAttributes - MEMORY_CACHED with ResourceLength as NULL the return status must be EFI_INVALID_PARAMETER.",
-                   L"%a:%d:Status - %r:Set Attributes - %LXh",
+                   L"%a:%d:Status - %r:Set Attributes - %lXh",
                    __FILE__,
                    (UINTN)__LINE__,
                    Status,
@@ -4600,7 +4600,7 @@ SetAttributes_Conf (
                    AssertionType,
                    gPciRootBridgeIoBBTestConformanceAssertionGuid066,
                    L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.SetAttributes - MEMORY_DISABLE with ResourceBase as NULL the return status must be EFI_INVALID_PARAMETER.",
-                   L"%a:%d:Status - %r:Set Attributes - %LXh",
+                   L"%a:%d:Status - %r:Set Attributes - %lXh",
                    __FILE__,
                    (UINTN)__LINE__,
                    Status,
@@ -4632,7 +4632,7 @@ SetAttributes_Conf (
                    AssertionType,
                    gPciRootBridgeIoBBTestConformanceAssertionGuid067,
                    L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.SetAttributes - MEMORY_DISABLE with ResourceLength as NULL the return status must be EFI_INVALID_PARAMETER.",
-                   L"%a:%d:Status - %r:Set Attributes - %LXh",
+                   L"%a:%d:Status - %r:Set Attributes - %lXh",
                    __FILE__,
                    (UINTN)__LINE__,
                    Status,
diff --git a/uefi-sct/SctPkg/TestCase/UEFI/EFI/Protocol/PciRootBridgeIo/BlackBoxTest/PciRootBridgeIoBBTestFunction_1.c b/uefi-sct/SctPkg/TestCase/UEFI/EFI/Protocol/PciRootBridgeIo/BlackBoxTest/PciRootBridgeIoBBTestFunction_1.c
index dcc9cfed56ee..351b6722ff52 100644
--- a/uefi-sct/SctPkg/TestCase/UEFI/EFI/Protocol/PciRootBridgeIo/BlackBoxTest/PciRootBridgeIoBBTestFunction_1.c
+++ b/uefi-sct/SctPkg/TestCase/UEFI/EFI/Protocol/PciRootBridgeIo/BlackBoxTest/PciRootBridgeIoBBTestFunction_1.c
@@ -381,7 +381,7 @@ PollMemCheckPoint (
                  StandardLib,
                  EFI_VERBOSE_LEVEL_DEFAULT,
                  L"PollMem env\n"
-                 L"%a:%d:RootBridgeIoWidth - %s,Mask - %LXh,TargetValue - %LXh,AlternateValue - %LXh",
+                 L"%a:%d:RootBridgeIoWidth - %s,Mask - %lXh,TargetValue - %lXh,AlternateValue - %lXh",
                  __FILE__,
                  (UINTN)__LINE__,
                  WidthCode[RootBridgeIoWidth],
@@ -416,7 +416,7 @@ PollMemCheckPoint (
                  AssertionType,
                  gTestGenericFailureGuid,
                  L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.Mem.Write - Write alternate value to memory",
-                 L"%a:%d:Status - %r, Alternate Value - %LXh",
+                 L"%a:%d:Status - %r, Alternate Value - %lXh",
                  __FILE__,
                  (UINTN)__LINE__,
                  Status,
@@ -488,7 +488,7 @@ PollMemCheckPoint (
                  AssertionType,
                  gPciRootBridgeIoBBTestFunction_1AssertionGuid001,
                  L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.PollMem() - Target address set the valid value before time out",
-                 L"%a:%d:Status - %r,Target Value - %LXh, Result - %LXh",
+                 L"%a:%d:Status - %r,Target Value - %lXh, Result - %lXh",
                  __FILE__,
                  (UINTN)__LINE__,
                  Status,
@@ -526,7 +526,7 @@ PollMemCheckPoint (
                  AssertionType,
                  gPciRootBridgeIoBBTestFunction_1AssertionGuid002,
                  L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.PollMem() - Target address has the valid value before calling",
-                 L"%a:%d:Status - %r, target value - %LXh, Result Value - %LXh",
+                 L"%a:%d:Status - %r, target value - %lXh, Result Value - %lXh",
                  __FILE__,
                  (UINTN)__LINE__,
                  Status,
@@ -560,7 +560,7 @@ PollMemCheckPoint (
                  AssertionType,
                  gTestGenericFailureGuid,
                  L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.Mem.Write() - write the alternate value again",
-                 L"%a:%d:Status - %r, alternate value - %LXh",
+                 L"%a:%d:Status - %r, alternate value - %lXh",
                  __FILE__,
                  (UINTN)__LINE__,
                  Status,
@@ -598,7 +598,7 @@ PollMemCheckPoint (
                  AssertionType,
                  gPciRootBridgeIoBBTestFunction_1AssertionGuid003,
                  L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.PollMem() - with delay equal 0",
-                 L"%a:%d:Status - %r,alternate value - %LXh,Result - %LXh",
+                 L"%a:%d:Status - %r,alternate value - %lXh,Result - %lXh",
                  __FILE__,
                  (UINTN)__LINE__,
                  Status,
@@ -637,7 +637,7 @@ PollMemCheckPoint (
                  AssertionType,
                  gPciRootBridgeIoBBTestFunction_1AssertionGuid004,
                  L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.PollMem() - with 5 seconds delay",
-                 L"%a:%d:Status - %r,alternate Value -%LXh, Result - %LXh",
+                 L"%a:%d:Status - %r,alternate Value -%lXh, Result - %lXh",
                  __FILE__,
                  (UINTN)__LINE__,
                  Status,
@@ -1388,7 +1388,7 @@ PollIoCheckPoint (
                  StandardLib,
                  EFI_VERBOSE_LEVEL_DEFAULT,
                  L"PollIo env\n"
-                 L"%a:%d:RootBridgeIoWidth - %s,Mask - %LXh,TargetValue - %LXh,AlternateValue - %LXh",
+                 L"%a:%d:RootBridgeIoWidth - %s,Mask - %lXh,TargetValue - %lXh,AlternateValue - %lXh",
                  __FILE__,
                  (UINTN)__LINE__,
                  WidthCode[RootBridgeIoWidth],
@@ -1422,7 +1422,7 @@ PollIoCheckPoint (
                  AssertionType,
                  gTestGenericFailureGuid,
                  L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.Io.Write - Write alternate value to Io space",
-                 L"%a:%d:Status - %r, Alternate Value - %LXh",
+                 L"%a:%d:Status - %r, Alternate Value - %lXh",
                  __FILE__,
                  (UINTN)__LINE__,
                  Status,
@@ -1496,7 +1496,7 @@ PollIoCheckPoint (
                  AssertionType,
                  gPciRootBridgeIoBBTestFunction_1AssertionGuid005,
                  L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.PollIo() - Target address set the valid value before time out",
-                 L"%a:%d:Status - %r,Target Value - %LXh, Result - %LXh",
+                 L"%a:%d:Status - %r,Target Value - %lXh, Result - %lXh",
                  __FILE__,
                  (UINTN)__LINE__,
                  Status,
@@ -1535,7 +1535,7 @@ PollIoCheckPoint (
                  AssertionType,
                  gPciRootBridgeIoBBTestFunction_1AssertionGuid006,
                  L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.PollIo() - Target address has the valid value before calling",
-                 L"%a:%d:Status - %r, target value - %LXh, Result Value - %LXh",
+                 L"%a:%d:Status - %r, target value - %lXh, Result Value - %lXh",
                  __FILE__,
                  (UINTN)__LINE__,
                  Status,
@@ -1569,7 +1569,7 @@ PollIoCheckPoint (
                  AssertionType,
                  gTestGenericFailureGuid,
                  L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.Io.Write() - write the alternate value again",
-                 L"%a:%d:Status - %r, alternate value - %LXh",
+                 L"%a:%d:Status - %r, alternate value - %lXh",
                  __FILE__,
                  (UINTN)__LINE__,
                  Status,
@@ -1607,7 +1607,7 @@ PollIoCheckPoint (
                  AssertionType,
                  gPciRootBridgeIoBBTestFunction_1AssertionGuid007,
                  L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.PollIo() - with delay equal 0",
-                 L"%a:%d:Status - %r,alternate value - %LXh,Result - %LXh",
+                 L"%a:%d:Status - %r,alternate value - %lXh,Result - %lXh",
                  __FILE__,
                  (UINTN)__LINE__,
                  Status,
@@ -1646,7 +1646,7 @@ PollIoCheckPoint (
                  AssertionType,
                  gPciRootBridgeIoBBTestFunction_1AssertionGuid008,
                  L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.PollIo() - with 5 seconds delay",
-                 L"%a:%d:Status - %r,alternate Value -%LXh, Result - %LXh",
+                 L"%a:%d:Status - %r,alternate Value -%lXh, Result - %lXh",
                  __FILE__,
                  (UINTN)__LINE__,
                  Status,
diff --git a/uefi-sct/SctPkg/TestCase/UEFI/EFI/Protocol/PciRootBridgeIo/BlackBoxTest/PciRootBridgeIoBBTestFunction_2.c b/uefi-sct/SctPkg/TestCase/UEFI/EFI/Protocol/PciRootBridgeIo/BlackBoxTest/PciRootBridgeIoBBTestFunction_2.c
index 0c7160775cf4..bd30e0875c06 100644
--- a/uefi-sct/SctPkg/TestCase/UEFI/EFI/Protocol/PciRootBridgeIo/BlackBoxTest/PciRootBridgeIoBBTestFunction_2.c
+++ b/uefi-sct/SctPkg/TestCase/UEFI/EFI/Protocol/PciRootBridgeIo/BlackBoxTest/PciRootBridgeIoBBTestFunction_2.c
@@ -1949,7 +1949,7 @@ CopyMem_Func (
                    AssertionType,
                    gPciRootBridgeIoBBTestFunction_2AssertionGuid015,
                    L"EFI_ROOT_BRIDGE_IO_PROTOCOL.CopyMem - Copying Data between non-overlapping regions.",
-                   L"%a:%d:Status - %r,Src Address -%LXh,Dest Address -%LXh,Count -%xh,Width -%s",
+                   L"%a:%d:Status - %r,Src Address -%lXh,Dest Address -%lXh,Count -%xh,Width -%s",
                    __FILE__,
                    (UINTN)__LINE__,
                    Status,
@@ -2029,7 +2029,7 @@ CopyMem_Func (
                    AssertionType,
                    gPciRootBridgeIoBBTestFunction_2AssertionGuid017,
                    L"EFI_ROOT_BRIDGE_IO_PROTOCOL.CopyMem - Copying Data between overlapping regions dest address > src address.",
-                   L"%a:%d:Status - %r,Src Address -%LXh,Dest Address -%LXh,Count -%xh,Width -%s",
+                   L"%a:%d:Status - %r,Src Address -%lXh,Dest Address -%lXh,Count -%xh,Width -%s",
                    __FILE__,
                    (UINTN)__LINE__,
                    Status,
@@ -2112,7 +2112,7 @@ CopyMem_Func (
                    AssertionType,
                    gPciRootBridgeIoBBTestFunction_2AssertionGuid019,
                    L"EFI_ROOT_BRIDGE_IO_PROTOCOL.CopyMem - Copying Data between overlapping regions with Dest < Src.",
-                   L"%a:%d:Status - %r,Src Address -%LXh,Dest Address -%LXh,Count -%xh,Width -%s",
+                   L"%a:%d:Status - %r,Src Address -%lXh,Dest Address -%lXh,Count -%xh,Width -%s",
                    __FILE__,
                    (UINTN)__LINE__,
                    Status,
@@ -4278,7 +4278,7 @@ AllocateBuffer_Func (
                      StandardLib,
                      EFI_VERBOSE_LEVEL_DEFAULT,
                      L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.AllocateBuffer - Status must be EFI_SUCCESS."
-                     L"%a:%d:Status - %r,MemoryType:%d,Attributes:%LXh",
+                     L"%a:%d:Status - %r,MemoryType:%d,Attributes:%lXh",
                      __FILE__,
                      (UINTN)__LINE__,
                      Status,
@@ -4304,7 +4304,7 @@ AllocateBuffer_Func (
                      StandardLib,
                      EFI_VERBOSE_LEVEL_DEFAULT,
                      L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.FreeBuffer - Status must be EFI_SUCCESS."
-                     L"%a:%d:Status - %r,MemoryType:%d,Attributes:%LXh",
+                     L"%a:%d:Status - %r,MemoryType:%d,Attributes:%lXh",
                      __FILE__,
                      (UINTN)__LINE__,
                      Status,
@@ -4528,7 +4528,7 @@ FreeBuffer_Func (
                      StandardLib,
                      EFI_VERBOSE_LEVEL_DEFAULT,
                      L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.AllocateBuffer - Status must be EFI_SUCCESS."
-                     L"%a:%d:Status - %r,MemoryType:%d,Attributes:%LXh",
+                     L"%a:%d:Status - %r,MemoryType:%d,Attributes:%lXh",
                      __FILE__,
                      (UINTN)__LINE__,
                      Status,
@@ -4553,7 +4553,7 @@ FreeBuffer_Func (
                      StandardLib,
                      EFI_VERBOSE_LEVEL_DEFAULT,
                      L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.FreeBuffer - Status must be EFI_SUCCESS."
-                     L"%a:%d:Status - %r,MemoryType:%d,Attributes:%LXh",
+                     L"%a:%d:Status - %r,MemoryType:%d,Attributes:%lXh",
                      __FILE__,
                      (UINTN)__LINE__,
                      Status,
@@ -4805,7 +4805,7 @@ GetAttributes_Func (
                  AssertionType,
                  gPciRootBridgeIoBBTestFunction_2AssertionGuid059,
                  L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.GetAttributes - current attributes must within Supported attributes",
-                 L"%a:%d:Current Attributes - %LXh, Supported Attributes - %LXh",
+                 L"%a:%d:Current Attributes - %lXh, Supported Attributes - %lXh",
                  __FILE__,
                  (UINTN)__LINE__,
                  CurrentAttributes,
@@ -4855,7 +4855,7 @@ GetAttributes_Func (
                  AssertionType,
                  gPciRootBridgeIoBBTestFunction_2AssertionGuid061,
                  L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.GetAttributes - two time returned Current attributes should equal",
-                 L"%a:%d:First Time Attributes - %LXh, Second Time Attributes - %LXh",
+                 L"%a:%d:First Time Attributes - %lXh, Second Time Attributes - %lXh",
                  __FILE__,
                  (UINTN)__LINE__,
                  CurrentAttributes,
@@ -4905,7 +4905,7 @@ GetAttributes_Func (
                  AssertionType,
                  gPciRootBridgeIoBBTestFunction_2AssertionGuid063,
                  L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.GetAttributes - two time returned supported attributes should equal",
-                 L"%a:%d:First Time Attributes - %LXh, Second Time Attributes - %LXh",
+                 L"%a:%d:First Time Attributes - %lXh, Second Time Attributes - %lXh",
                  __FILE__,
                  (UINTN)__LINE__,
                  SupportedAttributes,
@@ -5088,7 +5088,7 @@ SetAttributes_Func (
                    AssertionType,
                    gPciRootBridgeIoBBTestFunction_2AssertionGuid064,
                    L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.SetAttributes - Set Supported attribute that not need resource status must be EFI_SUCCESS",
-                   L"%a:%d:Status - %r,Set Attributes -%LXh,Supported Attributes -%LXh",
+                   L"%a:%d:Status - %r,Set Attributes -%lXh,Supported Attributes -%lXh",
                    __FILE__,
                    (UINTN)__LINE__,
                    Status,
@@ -5118,7 +5118,7 @@ SetAttributes_Func (
                AssertionType,
                gPciRootBridgeIoBBTestFunction_2AssertionGuid065,
                L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.SetAttributes - Supported attribute should really set",
-               L"%a:%d:Set Attribute - %LXh, Gotten Attributes - %LXh",
+               L"%a:%d:Set Attribute - %lXh, Gotten Attributes - %lXh",
                __FILE__,
                (UINTN)__LINE__,
                ThisAttribute,
@@ -5200,7 +5200,7 @@ SetAttributes_Func (
                    AssertionType,
                    gPciRootBridgeIoBBTestFunction_2AssertionGuid066,
                    L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.SetAttributes - Set Supported attribute which require a resource status must be EFI_SUCCESS",
-                   L"%a:%d:Status - %r,Set Attributes -%LXh,Supported Attributes -%LXh, Address -%LXh, Length -%lXh",
+                   L"%a:%d:Status - %r,Set Attributes -%lXh,Supported Attributes -%lXh, Address -%lXh, Length -%lXh",
                    __FILE__,
                    (UINTN)__LINE__,
                    Status,
@@ -5232,7 +5232,7 @@ SetAttributes_Func (
                    AssertionType,
                    gPciRootBridgeIoBBTestFunction_2AssertionGuid067,
                    L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.SetAttributes - Supported attribute should really set",
-                   L"%a:%d:Set Attribute - %LXh, Gotten Attributes - %LXh",
+                   L"%a:%d:Set Attribute - %lXh, Gotten Attributes - %lXh",
                    __FILE__,
                    (UINTN)__LINE__,
                    ThisAttribute,
diff --git a/uefi-sct/SctPkg/TestCase/UEFI/EFI/Protocol/PciRootBridgeIo/BlackBoxTest/PciRootBridgeIoBBTestStress.c b/uefi-sct/SctPkg/TestCase/UEFI/EFI/Protocol/PciRootBridgeIo/BlackBoxTest/PciRootBridgeIoBBTestStress.c
index 3812ba22ccd2..13df837d41a2 100644
--- a/uefi-sct/SctPkg/TestCase/UEFI/EFI/Protocol/PciRootBridgeIo/BlackBoxTest/PciRootBridgeIoBBTestStress.c
+++ b/uefi-sct/SctPkg/TestCase/UEFI/EFI/Protocol/PciRootBridgeIo/BlackBoxTest/PciRootBridgeIoBBTestStress.c
@@ -230,7 +230,7 @@ SetAttributes_Stress (
                      AssertionType,
                      gPciRootBridgeIoBBTestStressAssertionGuid031,
                      L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.SetAttributes - Set Supported attribute status must be EFI_SUCCESS",
-                     L"%a:%d:Status - %r,Set Attributes -%LXh,Supported Attributes -%LXh",
+                     L"%a:%d:Status - %r,Set Attributes -%lXh,Supported Attributes -%lXh",
                      __FILE__,
                      (UINTN)__LINE__,
                      Status,
@@ -254,7 +254,7 @@ SetAttributes_Stress (
                      AssertionType,
                      gPciRootBridgeIoBBTestStressAssertionGuid032,
                      L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.SetAttributes - Set Supported attribute the attributes should be really set.",
-                     L"%a:%d:Attributes before Set -%LXh,Set Attributes -%LXh,Attributes after Set -%LXh,Supported Attributes -%LXh",
+                     L"%a:%d:Attributes before Set -%lXh,Set Attributes -%lXh,Attributes after Set -%lXh,Supported Attributes -%lXh",
                      __FILE__,
                      (UINTN)__LINE__,
                      CurrentAttributes,
@@ -281,7 +281,7 @@ SetAttributes_Stress (
                      AssertionType,
                      gPciRootBridgeIoBBTestStressAssertionGuid033,
                      L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.SetAttributes - Set UnSupported attribute status must be EFI_UNSUPPORTED",
-                     L"%a:%d:Status - %r,Set Attributes -%LXh,Supported Attributes -%LXh",
+                     L"%a:%d:Status - %r,Set Attributes -%lXh,Supported Attributes -%lXh",
                      __FILE__,
                      (UINTN)__LINE__,
                      Status,
@@ -305,7 +305,7 @@ SetAttributes_Stress (
                      AssertionType,
                      gPciRootBridgeIoBBTestStressAssertionGuid034,
                      L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.SetAttributes - Set UnSupported attribute the attributes should remain unchanged",
-                     L"%a:%d:Attributes before Set -%LXh,Set Attributes -%LXh,Attributes after Set -%LXh,Supported Attributes -%LXh",
+                     L"%a:%d:Attributes before Set -%lXh,Set Attributes -%lXh,Attributes after Set -%lXh,Supported Attributes -%lXh",
                      __FILE__,
                      (UINTN)__LINE__,
                      CurrentAttributes,
diff --git a/uefi-sct/SctPkg/TestCase/UEFI/EFI/Protocol/PciRootBridgeIo/BlackBoxTest/PciRootBridgeIoBBTestSupport.c b/uefi-sct/SctPkg/TestCase/UEFI/EFI/Protocol/PciRootBridgeIo/BlackBoxTest/PciRootBridgeIoBBTestSupport.c
index 2e5369e14104..82a2ac90712f 100644
--- a/uefi-sct/SctPkg/TestCase/UEFI/EFI/Protocol/PciRootBridgeIo/BlackBoxTest/PciRootBridgeIoBBTestSupport.c
+++ b/uefi-sct/SctPkg/TestCase/UEFI/EFI/Protocol/PciRootBridgeIo/BlackBoxTest/PciRootBridgeIoBBTestSupport.c
@@ -1573,7 +1573,7 @@ EventNotifyWriteMem (
                                AssertionType,
                                gTestGenericFailureGuid,
                                L"EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL.Mem.Read - return status should be EFI_SUCCESS",
-                               L"%a:%d:status - %r,write value - %LXh, ReadValue - %LXh",
+                               L"%a:%d:status - %r,write value - %lXh, ReadValue - %lXh",
                                __FILE__,
                                (UINTN)__LINE__,
                                Status,
-- 
2.43.0

